.changes/unreleased/Added-20220321-162832.yaml

```diff
@@ -0,0 +1,3 @@
+kind: Added
```

Comment:
Is this a supported format? I feel like our process to roll up change logs might not like this. Based on the timestamp, it might be an older format.

Reply:
This part might be from before some more recent changes to our changie logic so yeah can remove